pricing-page-exercise/kadir/style.css

```diff
@@ -0,0 +1,232 @@
+* {
+    box-sizing: border-box;
+ }
+
+body {
+    margin: 0;
+    font-family: "Segoe UI",Roboto,"Helvetica Neue",Arial,sans-serif,"Apple Color Emoji","Segoe UI Emoji","Segoe UI Symbol";
+    }
+    
+
+.container_all{
+    width: 100vw;
+    height: 100%;
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-start;
+    align-items: center;
+}
+
+    .container_header{
+        width: 100%;
+        height: 80px;
+        padding:1%;
+        display:flex;
+        justify-content: space-between;
+        box-shadow: 0px 3px 5px -5px #111;     
+    }
+
+        .company{
+            height: 100%;
```

Comment:
You don't really need to set `height: 100%;` here. You use  `display:flex;` for `.container_header` and by default it make all the children stretch so they are aligned nicely.